### DISCUSSION TITLE
How to install this on seebox no root access?

### DISCUSSION THREAD

**Discussion by eskwire** [Taxel/PlexTraktSync#1952] [Q&A] [answered]
I try but have no idea how to. could someoene point me in the right direction?

I had another app before that was just drop in the plug inn folder on plex but seams this option its not avilable wit your app. HALP

**Comment by glensc** [comment, marked as answer]
same as https://github.com/Taxel/PlexTraktSync/discussions/1937:

What is seebox? what was that "another app"

Generic instructions are either install via docker image or install Python and use pipx.

what have you tried?

also, be nice and fix your typos.

**Comment by glensc** [comment]
User dissaparead. Closing

**Comment by eskwire** [comment]
Sorry still here. I wrote bad Seedbox. What happend its that on seedbox I dont have Admin access do can run ssh conection.
